### DISCUSSION TITLE
Floating URL Bar removed - Why?

### DISCUSSION THREAD

**Discussion by rapatel0** [zen-browser/desktop#284] [General]
I noticed that you removed the floating url bar commit:`6cf7fe9` 

I totally loved that feature. Why was it removed? too much maintenence?

**Comment by rapatel0** [comment]
@mauro-balades

**Comment by nocanoa** [comment]
Its here 
https://www.zen-browser.app/themes/83a641f7-eca9-4c0f-91af-45627bef0539

Will be working soon
